### DISCUSSION TITLE
Boilerplate or Starter Project for Beginners to get their feet wet

### DISCUSSION THREAD

**Discussion by JackieGable** [argyleink/open-props#167] [Q&A]
Just curious, is there a boilerplate or starter project for beginners to get their feet wet with this?  A simple template with a basic layout would sure be helpful. I've looked over the docs but I'm still not sure where or how to get started.

**Comment by argyleink** [comment]
Hi! Thanks for asking =)

I was hoping that's what [this section](https://open-props.style/#:~:text=Built%20with%20Open%20Props) did, but maybe it's not in the shape of an HTML file starter? Open Props doesnt really come with layouts and it's also just a bunch of values.. so it kinda doesnt do much out of the box.

I have an [open task to make github repo's](https://github.com/argyleink/open-props/issues/112) for some of the examples, would that be helpful here?

i'd love to help you kick start though, let me know what could be more helpful.
